lms/templates/edxnotes/edxnotes.html

```diff
@@ -26,7 +26,7 @@ <h1 class="page-title">
         <div class="wrapper-notes-search">
           <form role="search" action="${search_endpoint}" method="GET" id="search-notes-form" class="is-hidden">
             <label for="search-notes-input" class="sr">${_('Search notes for:')}</label>
-            <input type="search" class="search-notes-input" id="search-notes-input" name="note" placeholder="${_('Search notes for...')}">
+            <input type="search" class="search-notes-input" id="search-notes-input" name="note" placeholder="${_('Search notes for...')}" required aria-required="true">
```

Comment:
@talbs @clrux, are you sure I said that?  Was it before my morning coffee? If so, what I meant to say is that the aria-attribute is implied by the HTML5 required attribute.  aria-required only needs to be used when you use a role of input, select or textarea on some other element.  no need for redundancy here.